spec/controllers/idv/confirmations_controller_spec.rb

```diff
@@ -236,25 +245,20 @@ def index
       expect(@analytics).to have_logged_event(Analytics::IDV_DOWNLOAD_PERSONAL_KEY, success: true)
     end
 
-    it 'can be called separately from #show' do
+    it 'recovers pii with the code' do
+      get :show
       get :download
 
-      expect(response).to be_ok
+      code = response.body.chomp
 
-      code = subject.user_session[:personal_key]
-      expect(response.body).to eq(code + "\r\n")
+      expect(user.profiles.first.recover_pii(normalize_personal_key(code))).to be
     end
```

Comment:
this is adding a regression spec to make sure the downloaded code can decrypt the pii